samples/nextjs/src/lib/GraphQLClientFactory.ts

```diff
@@ -50,6 +50,9 @@ export default function GraphQLClientFactory(
       credentials: 'include',
       headers: {
         connection: 'keep-alive',
+        // Send the API key on the 'sc_apikey' header for authentication.
+        // This header is used for both Sitecore XM Edge schema and Sitecore Experience Edge.
```

Comment:
Is "Sitecore XM Edge schema" what it officially being called? Maybe having the word "Edge" in there could confuse people?

Reply:
@anastasiya29  Talking with Nick, going to use "Headless Services GraphQL endpoint" for now. Will add it to the list to confirm later.